demos/default/cache-handler.js

```diff
@@ -0,0 +1,19 @@
+const cache = new Map()
```

Comment:
This uses an in memory cache for the moment for debugging purposes to see key and values in the cache.